pages/linux/xclip.md

```diff
@@ -1,15 +1,32 @@
 # xclip
 
-> Copy STDIN to clipboard or print clipboard to STDOUT.
+> X11 clipboard manipulation tool, similar to `xsel`.
+> Handles the X primary and secondary selections, plus the system clipboard (`Ctrl + C`/`Ctrl + V`).
```

Comment:
What are these primary and secondary selections ?

Reply:
Unfortunately there's no space to explain these in the main description, but basically the primary selection is the one that gets pasted when you press the middle button/scroll wheel, if text has been selected anywhere (I think that's why it's called a selection area: any text selection is automatically stored there, IIUC).

The xsel page does mention the middle click to make this connection, but I didn't find a way to add it here without making the descriptions too long.